criu/config.c

```diff
@@ -420,6 +421,22 @@ static int parse_join_ns(const char *ptr)
 	return 0;
 }
 
+static int parse_file_validation_method(struct cr_options *opts, const char *optarg)
+{
+	if (!strcmp(optarg, "filesize"))
+		opts->file_validation_method = FILE_VALIDATION_FILE_SIZE;
+	else if (!strcmp(optarg, "buildid"))
```

Comment:
do we want to be able to specify filesize and buildid together?

Reply:
Regardless of what option the user picks (Build-id, checksum or just file size), the file size check is done first because if the file sizes aren't equal then there's no point in comparing the checksum or the build-id so it would be faster to immediately put out an error.